packages/config/src/layer2s/degate.ts

```diff
@@ -0,0 +1,265 @@
+      references: [
+        {
+          text: 'Forced Withdrawals - DeGate design doc',
+          href: 'https://github.com/degatedev/protocols/blob/degate_mainnet/Smart%20Contract%20Design.md#force-withdrawal',
+        },
+      ],
+    },
+    exitMechanisms: [
+      {
+        ...EXITS.REGULAR('zk', 'no proof'),
+        references: [
+          {
+            text: 'Withdraw - DeGate design doc',
+            href: 'https://github.com/degatedev/protocols/blob/degate_mainnet/Smart%20Contract%20Design.md#normal-withdrawal',
+          },
+        ],
+      },
+      {
+        ...EXITS.FORCED,
+        references: [
+          {
+            text: 'Forced Request Handling - DeGate design doc',
+            href: 'https://github.com/degatedev/protocols/blob/degate_mainnet/Smart%20Contract%20Design.md#force-withdrawal',
+          },
+          {
+            text: 'ExchangeV3.sol#L392 - DeGate source code, forceWithdraw function',
+            href: 'https://etherscan.io/address/0xe63602a9B3DFE983187525AC985Fec4F57B24eD5#code#F1#L392',
+          },
+        ],
+      },
+      {
+        ...EXITS.EMERGENCY(
+          'Withdrawal Mode',
+          'merkle proof',
+          forcedWithdrawalDelay,
+        ),
+        references: [
+          {
+            text: 'Forced Request Handling - DeGate design doc',
+            href: 'https://github.com/degatedev/protocols/blob/degate_mainnet/Smart%20Contract%20Design.md#exodus-mode',
+          },
+
+          {
+            text: 'ExchangeV3.sol#L420 - DeGate source code, withdrawFromMerkleTree function',
+            href: 'https://etherscan.io/address/0xe63602a9B3DFE983187525AC985Fec4F57B24eD5#code#F1#L420',
+          },
+        ],
+      },
+    ],
+  },
+  permissions: [
+    {
+      name: 'DefaultDepositContract Owner',
+      accounts: (() => {
+        const owner1 = discovery.getAddressFromValue(
+          'DefaultDepositContract',
+          'owner',
+        )
+        const owner2 = discovery.getAddressFromValue(
+          'LoopringIOExchangeOwner',
+          'owner',
+        )
+        const owner3 = discovery.getAddressFromValue('LoopringV3', 'owner')
+        const owner4 = discovery.getAddressFromValue('BlockVerifier', 'owner')
+
+        // making sure that the description is correct
+        assert(owner1 === owner2 && owner2 === owner3 && owner3 === owner4)
+
+        const permissionedAccount = discovery.formatPermissionedAccount(owner1)
+
+        // if it was updated, we should add multisig participants
+        assert(permissionedAccount.type === 'EOA')
+
+        return [permissionedAccount]
+      })(),
+      description:
+        'This address is the owner of the following contracts: LoopringIOExchangeOwner, LoopringV3, DefaultDepositContract, BlockVerifier. Can add or remove block submitters. Can change the forced withdrawal fee. Can change a way that balance is calculated per contract during the deposit, allowing the support of non-standard tokens.',
```

Comment:
I would specify here that there is a max fee value (and the actual value), it is an important difference compared to Loopring